### DISCUSSION TITLE
现在主页怎么只有一个温度了

### DISCUSSION THREAD

**Discussion by fatelove42** [immortalwrt/immortalwrt#1492] [General]
![image](https://github.com/user-attachments/assets/91b968d3-480f-4e80-a5cb-a741c8725e5e)
我记得原来是2个

**Comment by 1715173329** [comment]
很简单，为了保护你的隐私，省去打码的时间。

**Comment by xlighting2017** [comment]
这位使用 xiaomi_redmi-router-ax6s 的用户你好， 你的码打得不好

你的设备/驱动（mt7622/mt76），并没有提供标准的 "wifi温度读取接口" (/sys/class/thermal/phy*/hwmon*/temp1_input), 因此无法显示 wifi 温度

所以 “你记得” 究竟是 “哪个版本，什么硬件”， 我很好奇。

下次提出问题的时候，请附上完整的信息，方便别人定位。

**Reply by fatelove42** [reply]
![image](https://github.com/user-attachments/assets/a4ad9dfb-f718-4de8-b3d9-155ae38d07b8)
他又自己好了🤔
